hail/src/main/scala/is/hail/expr/types/virtual/TVariable.scala

```diff
@@ -10,10 +10,10 @@ import scala.reflect.ClassTag
 object TVariable {
   val condMap: Map[String, (Type) => Boolean] = Map(
     "numeric" -> ((t: Type) => t.isInstanceOf[TNumeric]),
-    "int32" -> ((t: Type) => t.isInstanceOf[TInt32]),
-    "int64" -> ((t: Type) => t.isInstanceOf[TInt64]),
-    "float32" -> ((t: Type) => t.isInstanceOf[TFloat32]),
-    "float64" -> ((t: Type) => t.isInstanceOf[TFloat64]),
+    "int32" -> ((t: Type) => t == TInt32),
```

Comment:
these can be TInt32() etc, to follow the empty constructor rule we just set for ourselves